app/experimenter/jetstream/client.py

```diff
@@ -217,4 +217,7 @@ def get_experiment_data(experiment):
 
     experiment_data["errors"] = errors_by_metric
 
-    return experiment_data
+    ret = {}
+    ret["v1"] = experiment_data
+
+    return ret
```

Comment:
What's wrong with `return {'v1': experiment_data}`?

Reply:
Nothing! I'll do that. The answer for how I got here is that I started with the ret object containing v1 and v2 and it was declared elsewhere. Realized that should happen in a future PR and didn't think about how this didn't make sense anymore :)